ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/EngineModeIT.scala

```diff
@@ -115,6 +114,8 @@ class EngineModeIT
           port = Port.Dynamic,
           engineMode = mode,
           seeding = Seeding.Weak,
+          // Ensure separate DBs per test
+          jdbcUrl = Some(s"jdbc:h2:mem:${UUID.randomUUID().toString};db_close_delay=-1"),
```

Comment:
Previously, this was ensured implicitly by the in-memory default setup of the sandbox classic